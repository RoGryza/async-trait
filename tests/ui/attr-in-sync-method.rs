use async_trait::async_trait;

#[async_trait]
pub trait Trait {
    #[async_trait(?Send)]
    fn method();
}

fn main() {}
