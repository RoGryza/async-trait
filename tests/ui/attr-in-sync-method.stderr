error: #[async_trait] is not allowed in sync methods
 --> tests/ui/attr-in-sync-method.rs:5:5
  |
5 |     #[async_trait(?Send)]
  |     ^^^^^^^^^^^^^^^^^^^^^
